smart-contracts/test/UPToken/swap.js

```diff
@@ -271,3 +271,64 @@ describe('Swapper UP / UDT', () => {
     })
   })
 })
+
+describe('Swapper UP / UDT with failing', () => {
```

Comment:
it is not very clear what is tested here. Could you add sth more explanatory instead of `with failing` ?